File: examples/file/delete/go/deleteFile.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/file"
)

// Delete a file by project name/path
func DeleteFile() {
	fileService := file.NewFileService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := fileService.Delete(&file.DeleteRequest{
		Path:    "/document/text-files/file.txt",
		Project: "examples",
	})
	fmt.Println(rsp, err)
}

File: examples/file/delete/node/deleteFile.js
const { FileService } = require("m3o/file");

// Delete a file by project name/path
async function deleteFile() {
  let fileService = new FileService(process.env.MICRO_API_TOKEN);
  let rsp = await fileService.delete({
    path: "/document/text-files/file.txt",
    project: "examples",
  });
  console.log(rsp);
}

deleteFile();
